app/redux/selectors/SetsSelectors.js

```diff
@@ -367,3 +367,37 @@ export const getSlowestDurationEver = (state, set) => {
 };
 
 export const getRevision = (state) => stateRoot(state).revision;
+
+// 1rm
+
+export const getExerciseVelocities = (state, exercise) => {   
+    const historySets = getHistorySets(state);
+    const workoutSets = getWorkoutSets(state);
+
+    const sets = historySets.concat(workoutSets);
+    let data = [];
+
+    sets.forEach((set) => {
+        if (set.exercise === exercise) {
+            data.push([set.weight, set.reps[0].data[2]]);
+        }       
+    });
+
+    return data;
+}
+
+export const get1rm = (state, exercise) => {
+    const lifts = getExerciseVelocities(state, exercise);
+    let maxWeight = lifts[0][0];
+    let slowestVel = lifts[0][1];
+
+    // highest lift
+    for (let i = 1; i < lifts.length; i++) {
```

Comment:
why are we skipping the 0th lift?

- [ ] add comments explaining what this entire loop is diong